sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_converters/_converter.py

```diff
@@ -36,6 +38,25 @@ def convert(
             subject=join_call_options.subject
             )
 
+class CreateCallRequestConverter(object):
```

Comment:
we don't need this CreateCallRequestConverter
@satyakonmsft do we need to have a overloading method for create_call_connection, named create_call? I don't think we need based on last week's overloading topic discussion.